BSP/STM32F413/Src/BSP_UART.c

```diff
@@ -202,7 +202,7 @@ void BSP_UART_Init(UART_t uartSelect) {
  * @param   str pointer to buffer string
  * @return  number of bytes that was read
  */
-uint32_t BSP_UART_ReadLine(UART_t uartSelect, char *str) {
+uint32_t BSP_UART_Read(UART_t uartSelect, char *str) {
```

Comment:
I'm curious: why the name change?

Reply:
We wrote the simulator with that function name, so it seemed silly to go back and change all instances of it to BPS's terminology.